dev-perl/text_bibtex/patches/text_bibtex-0.89.patchset

```diff
@@ -0,0 +1,47 @@
+From 4ad702485dfdfd7a259c90217bdd2f393674251d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Joachim=20Mairb=C3=B6ck?= <j.mairboeck@gmail.com>
+Date: Sun, 5 Jan 2025 23:42:32 +0100
+Subject: Haiku build support
+
+Set installation paths to Haiku conventions.
+
+Set LIBRARY_PATH for tests.
+
+diff --git a/Build.PL b/Build.PL
+index 3ec4ef8..488b985 100644
+--- a/Build.PL
++++ b/Build.PL
+@@ -87,6 +87,15 @@ if ( $^O =~ /mswin32/i ) {
+         unlink catfile( $libdir, $target );
+     }
+ }
++elsif ( $^O =~ /haiku/i ) {
++    $libdir =~ s/\bbin\b/lib/;
++    $hdrdir = $libdir;
++    $hdrdir =~ s!\blib\b!develop/headers!;
++    if ( $Config{archname} =~ /BePC/ ) {
++        $libdir .= "/x86";
++        $hdrdir .= "/x86";
++    }
```

Comment:
Is this check "good enough" for secondary arch support? It hard-codes the `/x86` suffix (i.e. the equivalent of `$secondaryArchSubDir` on x86) for x86 specifically.

If or when we get any other secondary architecture (with perl support), this will have to be revisited and improved.